src/components/AddItem/AddItem.jsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from "react";
+
+import ProductImg from "./components/ProductImg";
+import styles from "./styles/AddItem.module.css";
+
+export default function AddItemContent() {
+  const [productName, setProductName] = useState("");
+  const [description, setDescription] = useState("");
+  const [price, setPrice] = useState("");
+  const [tagInput, setTagInput] = useState("");
+  const [tags, setTags] = useState([]);
+  const [imagePreview, setImagePreview] = useState(null);
+
+  const isFormValid =
+    productName && description && price && tags.length > 0 && imagePreview;
+
+  const handleKeyDown = (e) => {
+    if (e.key === "Enter") {
+      e.preventDefault();
+      const trimmed = tagInput.trim();
+      if (trimmed && !tags.includes(trimmed)) {
+        setTags((prev) => [...prev, trimmed]);
+        setTagInput("");
+      }
+    }
+  };
+
+  const handleDelete = (targetTag) => {
+    setTags((prevTags) => prevTags.filter((tag) => tag !== targetTag));
+  };
+
+  return (
+    <main className={styles.container}>
+      <section className={styles.addContainer}>
+        <h1 className={styles.addTitle}>상품 등록하기</h1>
+        <button disabled={!isFormValid} className={styles.addBtn} type="submit">
+          등록
+        </button>
+      </section>
+
+      <section className={styles.contentContainer}>
+        <h2 className={styles.title}>상품 이미지</h2>
+        <ProductImg preview={imagePreview} setPreview={setImagePreview} />
+      </section>
+
+      <section className={styles.contentContainer}>
+        <h2 className={styles.title}>상품명</h2>
+        <input
+          className={styles.input}
+          placeholder="상품명을 입력해주세요"
+          value={productName}
+          onChange={(e) => setProductName(e.target.value)}
+        ></input>
+      </section>
+
+      <section className={styles.contentContainer}>
+        <h2 className={styles.title}>상품 소개</h2>
+        <textarea
+          className={styles.inputDescription}
+          placeholder="상품 소개를 입력해주세요"
+          value={description}
+          onChange={(e) => setDescription(e.target.value)}
+        ></textarea>
```

Comment:
💊 제안
label 태그로 하셔서 input과 연결하시면 더 좋을 것 같아요.

```suggestion
        <label htmlFor="description" className={styles.title}>상품 소개</label>
        <textarea
          id="description"
          className={styles.inputDescription}
          placeholder="상품 소개를 입력해주세요"
          value={description}
          onChange={(e) => setDescription(e.target.value)}
        />
```